cmd/link_package.go

```diff
@@ -21,7 +21,7 @@ func LinkPackage(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interr
 	here := "../.." + strings.Repeat("/..", depth)
 
 	gopath := Gopath()
-	if len(gopath) == 0 {
+	if gopath == "" {
```

Comment:
Makes sense. I have an unhealthy interest in microoptimizations, so I was
wondering if it was a performance thing.

On Friday, September 25, 2015, Giacomo Comitti <notifications@github.com>
wrote:

> In cmd/link_package.go
> <https://github.com/Masterminds/glide/pull/89#discussion_r40433214>:
>
> > @@ -21,7 +21,7 @@ func LinkPackage(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interr
> >  	here := "../.." + strings.Repeat("/..", depth)
> >
> >  	gopath := Gopath()
> > -	if len(gopath) == 0 {
> > +	if gopath == "" {
>
> I just think that comparing gopath against and empty string makes it
> clear that we are expecting a string from Gopath() instead of a more
> general slice.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Masterminds/glide/pull/89/files#r40433214>.
>


-- 
http://technosophos.com
https://github.com/Masterminds